02.calendar/cal.rb

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env ruby
+
+require "date"
+require "optparse"
+
+# 引数を読み込む
+opt = OptionParser.new
+option = {}
+opt.on("-y") {|x| option[:y] = x}
+opt.on("-m") {|x| option[:m] = x}
```

Comment:
```suggestion
opt.on("-y") {|y| option[:y] = y}
opt.on("-m") {|m| option[:m] = m}
```

xに意味はないと思うので、それよりは多少意味を持った変数名を使いましょう

Reply:
:yはyear
:mはmonth
の変数名に変更しました。